packages/safety-suite-api/src/api/userManagement/users.ts

```diff
@@ -0,0 +1,74 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiResponse, CustomerSpecificQuery, Id} from '../../types';
+import {User} from './types';
+
+export interface GetUsersQuery extends Partial<CustomerSpecificQuery> {
+  /**
+   * Filter users by these ids.
+   */
+  ids?: Id[];
```

Comment:
Endpoint which accepts `ids` parameter is also a Machine-to-Machine endpoint and it should be removed.
We still can use this endpoint, but without `ids` parameter.
I will remove this endpoint from API Reference in scope of this PR: https://github.com/idelic-inc/api-reference/pull/34